### ISSUE TITLE
Enable attribute display mode of a relation

### ISSUE THREAD

**Issue by jakubklimek** [mff-uk/dataspecer#545]
In DCAT, there is `dcat:accessURL` which behaves like an attribute (of a Distribution), but has the `rdfs:Resource` range, as the values are IRIs of RDF resources, not literals with `xsd:anyURI` datatype, which makes it an object property, and this now needs to be modeled as a relation.

One possible solution would be to really model it as a relation and "switch it" to attribute view.
Another solution would be to add `rdfs:Resource` as another datatype and add a special behavior to it (creating a relation switched to attribute view). This could be also extended to all classes, maybe offered as an alternative to data types.

These options may have some side-effects connected to visibility of relations, classes etc. which need to be investigated.

**Comment by jakubklimek** [comment]
This feature could be controlled by additional button in the list of attributes and relationships.

**Comment by skodapetr** [comment]
@jakubklimek Should this also move the representation in the catalog between associations/relationships and attributes?

**Comment by jakubklimek** [comment]
@skodapetr No. The catalog primarily shows the semantic level, not the visual level (except for what is visible or not). Therefore, catalog should display the semantic property type, not the visual one.

**Comment by jakubklimek** [comment]
The third option would be to display it as both the attribute and relation.

Use case for switching in general, and for displaying it as both:

![Image](https://github.com/user-attachments/assets/874acde9-9182-4711-be63-f79508460ebc)
vs

![Image](https://github.com/user-attachments/assets/d37170a4-5042-4e38-a891-882e08a0fbf3)
